style.css

```diff
@@ -19,4 +19,117 @@ h1,h2,h3,h4,h5,h6{
 }
 p{
     font-family: 'Open Sans', sans-serif;
-}
\ No newline at end of file
+}
+header {
+    display: flex;
+    position: fixed;
+    justify-content: space-around;
+    position: absolute;
```

Comment:
why you used `position` property twice, remove this line you don't need it. since you gave it a fixed position.
Also Don't forget to add `top: 0` property so the header stays on the top of your page